SPECS/xapi.spec

```diff
@@ -118,6 +118,7 @@ Patch1023: 0023-CA-409482-Using-computed-delay-for-RRD-loop.patch
 Patch1024: 0024-CA-411679-Runstate-metrics-return-data-over-100.patch
 Patch1025: 0025-xcp-rrdd-change-the-code-responsible-for-filtering-o.patch
 Patch1026: 0026-rrdd-Avoid-missing-aggregation-of-metrics-from-newly.patch
+Patch1027: 0027-CA-407370-Use-remote.conf-for-customer-rsyslog-forwa.patch
```

Comment:
It's grouped with RRD fixes but it isn't one. A mention of the upstream version it will be one might be useful too.

Reply:
Fixed